bpf/Makefile

```diff
@@ -24,11 +24,13 @@ BPFTESTDIR := test/
 
 ALIGNCHECKER = bpf_alignchecker.o
 PROCESS = bpf_execve_event.o bpf_execve_event_v53.o bpf_fork.o bpf_exit.o bpf_generic_kprobe.o \
+	  bpf_execve_bprm_commit_creds.o bpf_execve_bprm_commit_creds_v53.o \
```

Comment:
Is there any reason to have 3 different versions of this program? I cannot find any differences related to the kernel version. Why not just have `bpf_execve_bprm_commit_creds.o` for all kernel versions?

Reply:
Sure, thanks